charts/seed-bootstrap/charts/elastic-kibana-curator/values.yaml

```diff
@@ -0,0 +1,24 @@
+global:
+  elasticsearchPorts:
+    db: 9200
+    transport: 9300
+kibanaPort: 5601
+
+ingress:
+  enabled: true
+  host: k.seed-1.example.com
+  # admin : admin base64 encoded
+  basicAuthSecret: YWRtaW46JGFwcjEkSWRSaVM5c3MkR3U1MHMxaGUwL2Z6Tzh2elE4S1BEMQ==
+
+elasticsearchReplicas: 1
+kibanaReplicas: 1
+
+elasticsearchVolumeSizeGB: 30
+# Should be 2/3 of the above:
+curatorCutoverSizeGB: 20
```

Comment:
if it has to be 2/3 of the `elasticsearchVolumeSizeGB`, why is it configurable and not computed on the fly?

Reply:
Because I could not find out how to do it. Can you give me a clue?